### PR TITLE
📝 Try and encourage people to take Prometheus issues upstream.

### PR DESCRIPTION
**Pull request by coderanger**

### CODE REVIEW

**Review by paulfantom — APPROVED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Feel free to ask questions in #prometheus-operator on Kubernetes Slack!
 Note: This repository is about prometheus-operator itself, if you have questions about:
 - helm installation, go to https://github.com/helm/charts repository
```
Comment (paulfantom) [inline .github/ISSUE_TEMPLATE/bug.md:12]: We should change that link too. prometheus-operator helm chart is now hosted in https://github.com/prometheus-community/helm-charts

**Review by brancz — COMMENTED** [review]
I do want to make sure we don't loose a key position in the Prometheus/Kubernetes intersection. People ask Prometheus questions here because we've created an environment that is much more approachable, while Prometheus is generally viewed as a rather hostile encounter. I would encourage us to not to be too harsh to people, especially those that are just entering this community (if we treat new people nicely, they are more likely to come back and contribute/open issues/etc.).

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ Feel free to ask questions in #prometheus-operator on Kubernetes Slack!
...
 - kube-prometheus setup, go to https://github.com/prometheus-operator/kube-prometheus
+- prometheus basic configuration, go to https://prometheus.io/docs/introduction/overview/
+- bugs in prometheus, go to https://github.com/prometheus/prometheus/issues
```
[the remark this answers is not in the document]
Comment (metalmatze) [reply .github/ISSUE_TEMPLATE/bug.md:15]: I agree. It's long been that way and I think it often also helped reduce the number of issues opened against Prometheus to the ones really intended to go there.

**Pull request closed without merging**